### PR TITLE
Store - style trash pickup

### PR DESCRIPTION
**Pull request by jameskoster**
Just a few odds and ends that need tidying up;

- [x] Trash action icon styling for shipping zones doesn't match products / promos etc
- [x] Order status accessibility - I'm not convinced the colors meet WCAG AA guidelines for contrast
- [x] Print and fulfil button alignment https://cloudup.com/cqB9YXxtKdJ
- [x] Edit order - ragged right alignment https://cloudup.com/c2iRPRzd_hO
- [x] "Applies" dropdown margin in promos https://cloudup.com/c_lWsqqKSaY
- [x] Orders save button label = save. Promos + products = update. Should be consistent.
- [x] Margin beneath "Payment authorisation" radios in PayPal settings (same applies for all payment methods) https://cloudup.com/cEVXKFyupx7
- [x] Cash on delivery dialog is unnecessarily narrow https://cloudup.com/cCOAxTq6B91 let's make all methods the same
- [x] Edit address link color in shipping & tax settings https://cloudup.com/clIv4LElp-q
- [x] Selecting a location in Zones changes the dialog width https://cloudup.com/cWpq75Ph-WP
- [x] Order actions on smaller screens https://cloudup.com/cOEeKYa3G5R
- [x] Instances of "Save and Finish" vs "Save & Finish"

### CODE REVIEW

File: client/extensions/woocommerce/app/settings/shipping/save-button.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ class ShippingSettingsSaveButton extends Component {
 				</Button>
 			) : null;
 		}
-		const label = wcsEnabled ? translate( 'Save and finish' ) : translate( "I'm Finished" );
+		const label = wcsEnabled ? translate( 'Save & finish' ) : translate( "I'm Finished" );
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/shipping/save-button.js]: Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Finished!' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/store-styles)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by kellychoffman** [comment]
> Trash action icon styling for shipping zones doesn't match products / promos etc
> Order status accessibility - I'm not convinced the colors meet WCAG AA guidelines for contrast
> Print and fulfil button alignment https://cloudup.com/cqB9YXxtKdJ
> "Applies" dropdown margin in promos https://cloudup.com/c_lWsqqKSaY
> Orders save button label = save. Promos + products = update. Should be consistent.
> Margin beneath "Payment authorisation" radios in PayPal settings (same applies for all payment methods) https://cloudup.com/cEVXKFyupx7
> Cash on delivery dialog is unnecessarily narrow https://cloudup.com/cCOAxTq6B91 let's make all methods the same
> Edit address link color in shipping & tax settings https://cloudup.com/clIv4LElp-q
> Selecting a location in Zones changes the dialog width https://cloudup.com/cWpq75Ph-WP
> Order actions on smaller screens https://cloudup.com/cOEeKYa3G5R
> Instances of "Save and Finish" vs "Save & Finish"

👍 Looks groovy. 

> Edit order - ragged right alignment https://cloudup.com/c2iRPRzd_hO

Not sure this is an improvement - the Total column isn't aligned now.

**Comment by jameskoster** [comment]
That's a good point. Not sure the column really carries through the button row beneath though?

Here's an alternative, but that massive space feels weird to me;

space

**Comment by kellychoffman** [comment]
Good point about the button. 

Is this weird? 
screen shot 2018-01-03 at 9 18 51 am

**Comment by jameskoster** [comment]
I think that's better. Do you think they _need_ to be buttons?

buttons

**Comment by kellychoffman** [comment]
In the edit order view, no. But the add-product should be a button in the create order view. @ryelle: is that possible to do?

**Comment by ryelle** [comment]
Yeah, they can both become borderless buttons on edit & primary+border on add (but technically they need to stay buttons for interactions).

**Comment by kellychoffman** [comment]
Awesome, thanks @ryelle. Agree about buttons (borderless buttons is fine, right?) 

Should I make a separate issue for this or should the change go in this PR?

**Comment by jameskoster** [comment]
Can we do that separately? I'd really like to get this merged asap.

**Comment by kellychoffman** [comment]
Agree. Let’s do it separately.

**Pull request merged**